### ISSUE TITLE
Can we get a non-ES6 no-module dist package for npm?

### ISSUE THREAD

**Issue by FlorentMasson** [chrisrzhou/three-glow-mesh#1]
Thanks! Amazing time-saver

**Comment by chrisrzhou** [comment]
Hey @FlorentMasson, thanks for checking the package out!  I believe the non-module `dist` package should be the default package:

https://github.com/chrisrzhou/three-glow-mesh/blob/master/package.json#L5

If your project uses ES6 module, it should automatically use modules:

https://github.com/chrisrzhou/three-glow-mesh/blob/master/package.json#L6

Unless I'm not understanding the exact question.  I'm not super knowledgeable on package building and know the bare minimum from what I've learned online, so happy to learn how to address the issue.

**Comment by FlorentMasson** [comment]
This file (dist/index.js) does not work in a non-ES6 setup (ie when you use node_modules/three/build/three.js instead of node_modules/three/build/three.module.js)

![image](https://user-images.githubusercontent.com/9975455/76837635-2c0ea700-6833-11ea-8ba4-c490697e9cf3.png)

To reproduce, just add this to your html file and it will break
`    <script type="text/javascript" src="node_modules/three-glow-mesh/dist/index.js"></script>
`

**Comment by chrisrzhou** [comment]
@FlorentMasson, it's been awhile since I got back on this, but good news is I've been getting more experience with bundling packages.  I think I can make this work.  Will probably look into this this weekend if it is still relevant.

**Comment by chrisrzhou** [comment]
Hey @FlorentMasson, just shipped `v0.1.2` that uses `microbundle` for building the package.  If you need access to a non-module package, you can use the `dist/index.umd.js` file.  I'm not very well-versed in different build packages, but I tested that setting the following up does not throw an error:

```html
<html>
    <head>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r106/three.min.js"></script>
        <script src="./index.umd.js"></script>
    </head>
    <body>Test three-glow-mesh</body>
</html>
```

Using the `index.js` (CommonJS), `index.module.js` (ES module) will throw varying errors you are referring to.

Closing the issue but ping me if this isn't resolved on your side.
